lib/engines/content_block_manager/app/controllers/content_block_manager/content_block/editions/workflow_controller.rb

```diff
@@ -73,15 +73,26 @@ def review
   def confirmation
     @content_block_edition = ContentBlockManager::ContentBlock::Edition.find(params[:id])
 
+    confirmation_copy
+
+    render :confirmation
+  end
+
+  def confirmation_copy
```

Comment:
that's a good shout, was thinking that it was a lot of annoying logic - I'll do it in another PR just to avoid holding this one up

Reply:
ok, I've tried for a bit to put this into a helper method in a clean way - but because it needs the edition and the is_scheduled param it makes it a bit more complicated than I thought (would the view have to pass those variables to the helper method, because controllers aren't meant to call helpers according to the docs?) in javascript land I would just stick in a utils, not sure what to do here that wouldn't make it more complicated...

Reply:
One approach could be using the presenter pattern, have a class in `app/presenters` that you initialize in the controller, then pass to the view